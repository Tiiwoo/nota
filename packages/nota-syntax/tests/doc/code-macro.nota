%let f = macro{**{#1}.{#2}**}
#f{a}{b}