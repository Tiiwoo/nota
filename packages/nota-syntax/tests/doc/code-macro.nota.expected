[
  ...(() => {
    let f = (...args) => ["**{", args[0], "}.{", args[1], "}**"];

    return [null, el("p", {}, f(["a"], ["b"]))];
  })(),
];